app/config.py

```diff
@@ -267,6 +267,7 @@ class Config(object):
     AWS_SES_SECRET_KEY = os.getenv("AWS_SES_SECRET_KEY")
     AWS_PINPOINT_REGION = os.getenv("AWS_PINPOINT_REGION", "us-west-2")
     AWS_PINPOINT_SC_POOL_ID = os.getenv("AWS_PINPOINT_SC_POOL_ID", None)
+    AWS_PINPOINT_DEFAULT_POOL_ID = os.getenv("AWS_PINPOINT_DEFAULT_POOL_ID", None)
```

Comment:
I wonder if we really want `None` here as a fallback default, or rather prevent the server from booting up to prevent a misconfiguration. It depends how `OriginationIdentity` set to `None` in the `send_text_message` call behaves.

Reply:
Ok reading the code further, it seems that a `None` value would route the SNS implementation to get used instead. I got this right?

Reply:
yeah exactly. A `None` value will break the Pinpoint send, so right now we route to SNS. In the future we may want to throw an error instead (ie when we insist on Pinooint for everything). Right now it allows production to ignore pinpoint entirely since we don't have it set there yet.